CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Secondary Nav Scroll Restoration
```

Comment:
can we update this to be a bit more descriptive for those less familiar with the project

Reply:
Let me know if this description does not make sense: https://github.com/cerner/terra-dev-site/pull/217/commits/da753b47dc078f307181297bd340199e3d8f04f5

Reply:
@gabeparra01 makes sense to me!